package-lock.json

```diff
@@ -30,6 +30,46 @@
         "js-tokens": "^4.0.0"
       }
     },
+    "@eslint/eslintrc": {
+      "version": "1.3.0",
+      "resolved": "https://trynow-352765517868.d.codeartifact.us-east-1.amazonaws.com:443/npm/trynow-nodejs/@eslint/eslintrc/-/eslintrc-1.3.0.tgz",
```

Comment:
These are internal TryNow CodeArtifact URLs that need to be replaced by the NPM ones. You'll need to do the same as you did in #1

Reply:
Forkin registry. I'll clear this up once I add the object curly spacing!